src/legacy/ui/public/validated_range/is_range_valid.test.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { isRangeValid } from './is_range_valid';
+
+const formatMessageMock = () => {
+  return '';
+};
```

Comment:
nit: It would be really easy to also test that you're returning the correct error messages for invalid ranges, by just confirming the message has the expected ID:

```javascript
const mockFormatMessage = jest.fn().mockReturnValue('');
beforeEach(() => jest.mockFormatMessage.mockClear());

...

const errId = mockFormatMessage.mock.calls[0].id;
expect(errId.includes('upperValidErrorMessage')).toBe(true);
```